packages/styled-ui-docs/components/ThemeParser.jsx

```diff
@@ -19,11 +19,19 @@ const jsonStringify = (obj, indent) => {
 const ThemeParser = ({ theme, mode, ...props }) => {
   const themes = useTheme();
   const colorPalette = getColorPalette(mode);
-  const token = themes[theme] || colorPalette[theme];
   const indent = !!mode;
+  let token = themes[theme] || colorPalette[theme];
   if (!token) {
     return 'Theme field not found';
   }
+  if (theme === 'space' || theme === 'sizes') {
+    token = Object.keys(token)
+      .filter(key => !(key.toString().match(/\q|h/g))) // Filter strings matching 'g' or 'h' like '1q', '1h'
```

Comment:
`'q'` was incorrectly spelled as `'g'` in the comment

`Filter strings matching 'q' or 'h' like '1q', '1h'`

Reply:
How about using this regular expression?
```js
toString().match(/[qh]$/);
```

The "g" flag is not necessary here for testing against all possible matches in a string.